Jack/Script.fsx

```diff
@@ -112,6 +112,30 @@ Property.print <| property {
 }
 
 //
+// Loops
+//
+
+Property.print <| property {
+    for x in "abcd" do
+        // Custom operations (i.e. counterexample) can't be used in computation
+        // expressions which have control flow :( we can fake it using return!
+        // however.
+        return! Property.counterexample (sprintf "x = %A" x)
+
+        // Note, return can be used multiple times, its a bit like 'assert'.
+        return x <> 'w'
+        return x <> 'z'
```

Comment:
@jystic I tend to not like this either, but haven't yet spend enough time on this, so I don't have some useful feedback for the time being.